.parcelrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@parcel/config-default",
   "transformers": {
-    "*.{ts,tsx}": ["@parcel/transformer-typescript-tsc"]
+    "*.{ts,tsx}": ["@parcel/transformer-typescript-tsc"],
+    "*.svg": ["...", "@parcel/transformer-svg-react"]
```

Comment:
Curious about this. Were the `svg` causing build issues?

Reply:
There were no build issues, but the links to the SVG file that the core code generated when run inside enterprise just pointed at e.g. `file:///success.3e193089.svg` instead of being resolved on the server, so they wouldn't load.

The actual code generated by `make ui-lib` contained
```
var $72fdb1cb9e084dbe$exports = {};
$72fdb1cb9e084dbe$exports = new URL("success.3e193089.svg", "file:" + __filename
).toString();
```
which looks to me like parcel does nothing extra to indicate that the URL needs to be resolved, and react-scripts doesn't think to try to read it from the local directory - and I suspect there's some way to make it work this way cleanly through various faffing around with bundlers, and also I suspect that'd take longer than I expect.

So I just grabbed one of the other approaches in https://parceljs.org/languages/svg/#importing-as-a-react-component